### DISCUSSION TITLE
How do I reopen a presentation I worked on before?

### DISCUSSION THREAD

**Discussion by maulynvia** [senshu/Sozi#567] [Q&A]
This must be a very basic question. When I launch Sozi, I have to select an svg file, but this doesn't contain any of the presentation info. How do I open it so it has all the presentation logic is there and can be worked on further?

(Love Sozi, by the way, great to get away from conventional slides)

**Comment by aumouvantsillage** [comment]
By design, Sozi does not alter the original SVG document. It saves the presentation data and the final result in separate files.

When you create a Sozi presentation from an SVG document, the editor will create a data file (with extension `.json`) in the same folder.
If you reopen the same SVG, the presentation editor will automatically reload the presentation data from the `.json` file.

**Reply by hoshi411** [reply]
Which kind of makes the "project folder" idea seem pleasing.

I imagine a sozi project folder which would include fonts folder and external css file.
Some might be interested in a scripts folder as well which would allow to add external javascript libraries and scripting.

Sozi would simply generate the links to these files automatically so that the users would not need to manually add them every time sozi saves.

But don't get me wrong. Sozi is amazing as it is. And i am loving it.
